lib/src/smtp/internal_representation/ir_content.dart

```diff
@@ -173,7 +173,7 @@ class _IRContentAttachment extends _IRContent {
 enum _IRTextType { plain, html }
 
 class _IRContentText extends _IRContent {
-  String? _text;
+  String _text = '';
```

Comment:
This should fix the bug mentioned in https://github.com/kaisellgren/mailer/issues/167